projects/vdk-plugins/vdk-oracle/src/vdk/plugin/oracle/oracle_configuration.py

```diff
@@ -122,3 +126,8 @@ def add_definitions(config_builder: ConfigurationBuilder):
             "Before setting this follow instruction in "
             "https://python-oracledb.readthedocs.io/en/latest/user_guide/initialization.html#enablingthick ",
         )
+        config_builder.add(
+            key=ORACLE_INGEST_BATCH_SIZE,
+            default_value=1000,
```

Comment:
How did you decide on using 1000 as a default batch size? Isn't this a bit too big? Is it good performance-wise?
I saw this recommendation from the Oracle docs for 50-100: https://docs.oracle.com/cd/E11882_01/java.112/e16548/oraperf.htm#JJDBC28754

Reply:
Thanks for pointing this out, I'll change the default.